mslib/mswms/mss_plot_driver.py

```diff
@@ -533,6 +534,9 @@ def plot(self):
         else:
             resolution = (-1, -1)
 
+        if self.return_format not in ("image/png", "text/xml"):
```

Comment:
I'm not sure if we have to name it by a prefix of "return_"  Also we could use the name mimetype which links it to e.g. https://wiki.selfhtml.org/wiki/MIME-Type/%C3%9Cbersicht

Reply:
maybe. I did not invent the name. As this goes to stable, I try to keep changes minimal. These are already more changes than I intended to do.